api/evaluation/evaluation.proto

```diff
@@ -0,0 +1,133 @@
+ *  \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
+ *
+ * This file is part of Clouditor Community Edition.
+ */
+ syntax = "proto3";
+
+ package clouditor;
+
+ import "google/api/annotations.proto";
+ import "google/protobuf/timestamp.proto";
+ import "api/assessment/assessment.proto";
+ import "api/orchestrator/orchestrator.proto";
+ import "tagger/tagger.proto";
+
+
+ option go_package = "clouditor.io/clouditor/api/evaluation";
+
+ // This service represents the Evaluation Interface
+ service Evaluation {
+    // StartEvaluation evaluates all assessment results of an service based on its Target of Evaluation (binding of a cloud service to a catalog). The assessment results are evaluated regarding the contol ID.
+    rpc StartEvaluation(StartEvaluationRequest) returns (StartEvaluationResponse) {
+    option (google.api.http) = {
+        post : "/v1/evaluation/evaluate/{category_name}/{control_id}/start"
+        body : "toe"
+      };
+     } 
+
+    // StopEvaluation stop the evaluation for the given Cloud Service
+    rpc StopEvaluation(StopEvaluationRequest) returns (StopEvaluationResponse) {
+    option (google.api.http) = {
+        post : "/v1/evaluation/evaluate/{category_name}/{control_id}/stop"
+        body : "toe"
+        };
+    } 
+
+    // List all evaluation results. Part of the public API, also exposed as REST.
+    rpc ListEvaluationResults(ListEvaluationResultsRequest)
+        returns (ListEvaluationResultsResponse) {
+        option (google.api.http) = {
+            get : "/v1/evaluation/results"
+        };
+    }
+}
+
+message ListEvaluationResultsRequest {
+    string filtered_cloud_service_id = 1;
+  
+    int32 page_size = 10;
+    string page_token = 11;
+    string order_by = 12;
+    bool asc = 13;
+  }
+  message ListEvaluationResultsResponse {
+    repeated EvaluationResult results = 1;
+    string next_page_token = 2;
+  }
+
+message StartEvaluationRequest {
+    TargetOfEvaluation toe = 1;
+    // The category the control ID belongs to 
+    string category_name = 3; 
+    // The control ID
+    string control_id = 4; 
+}
+
+message StartEvaluationResponse {
+    bool status = 1; 
+    string status_message = 2; 
+}
+
+message StopEvaluationRequest {
+    TargetOfEvaluation toe = 1; 
+    // The category the control ID belongs to 
+    string category_name = 2; 
+    // The control ID
+    string control_id = 3;
+}
+
+message StopEvaluationResponse {}
+ 
+// A evaluation result resource, representing the result after evaluating the cloud service with a specific control
+message EvaluationResult {
+    // Evaluation result id
+    string id = 1; 
+
+    enum EvaluationStatus {
+        STATUS_UNSPECIFIED = 0;
+        COMPLIANT = 1;
+        NOT_COMPLIANT = 2;
+        PENDING = 3;
+    }
+    
+    // Evaluation status
+    EvaluationStatus status = 2;
+    
+    // Reference to the control the evaluation was based on
+    string control = 3;
```

Comment:
do we need the category name here as well?

Reply:
Oh, I was in the wrong line. I don 't know if we need the category name, but we can add it.